extension/app/highlighter.js

```diff
@@ -1,36 +1,48 @@
 /* jshint esversion: 6 */
 
-define(['../var/document', '../var/languages'], function(document, languages) {
+define(['../var/document', '../var/languages'], (document, languages) => {
     'use strict';
 
     const INTERVAL = 50; // Interval in milliseconds.
 
+    return {
+        init() {
+            insertStyles();
+            waitForRender('.diff-container').then(() => {
+                prepareLanguageClasses();
+                addCodeTagToElements('.source');
```

Comment:
I forgot to explain above, but the reason I changed it to pass '.source' is that later we can pass '.source:not([class*="language"])', to create the `<code>` elements for the elements that are not highlighted yet.